solidity-v1/dashboard/src/hooks/useAutoConnect.js

```diff
@@ -27,11 +30,11 @@ const useAutoConnect = () => {
   const isExactRoutePath = useIsExactRoutePath()
   const [injectedTried, setInjectedTried] = useState(false)
 
-  const isWalletFromUrlSameAsInMetamask = useCallback(
-    (metamaskAccounts) => {
+  const isWalletFromUrlSameAsInProvider = useCallback(
+    (providerAccount) => {
```

Comment:
We are passing here array of accounts returned from `getAccounts()` method so I think we should keep it as plural:

```suggestion
    (providerAccounts) => {
```